plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout.spec.js

```diff
@@ -133,7 +133,9 @@ test.describe( 'Checkout page', () => {
 			'1'
 		);
 		let totalPrice = await page.getByRole( 'row', { name: 'Total' } ).last().locator( 'td' ).textContent();
+		console.log( `Total Price: ${ totalPrice }` );
```

Comment:
The changes in this file aren't going to fix the failures, but I'm just trying to get a little more information about why they're failing.